tdct/deconvolution.py

```diff
@@ -0,0 +1,864 @@
+import sys
+import os
+
+import tifffile
+if getattr(sys, 'frozen', False):
+    # programm runs in a bundle (pyinstaller)
+    execdir = sys._MEIPASS
+else:
+    execdir = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(execdir)
+
+import numpy as np
+from PyQt5 import QtWidgets
+from . import clrmsg
+from . import TDCT_debug
+
+try:
+    import tifffile as tf
+except:
+    sys.exit("Please install tifffile, e.g.: pip install tifffile")
+
+debug = TDCT_debug.debug
+
+import scipy.signal
+import scipy.fft
+
+#Internal helper class to handle progress bar
+class _progrBarHandle():
+    def __init__(self, qtprocessbar, niter):
+        self.iter=0
```

Comment:
I ran your code through the "black" code formatter. I find this really useful for making all the code formatting consistent. And it fixes little things like this where you have no spaces around the "=" and below where you have spaces around the "=".

https://black.readthedocs.io/en/stable/